tests/SMS77Test.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Okay. You are right. Does not make sense in a unit test, as those should be dependent only on our system.
But for a SMS lib I think such API tests are very useful. So actually it is an integration test. Should we create a separate `IntegrationTest.php`? Actually the integration test could cove all of the Drivers and just be switchable to choose the one(s) that are of interest in a specific case.